keyboards/zj68/zj68.h

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2019 Collin Diekvoss <cdiekvoss@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef ZJ68_H
```

Comment:
in order to clean up the include guards we have been using `#pragma once` instead. 

```suggestion
#pragma once
```